app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -129,6 +129,22 @@ export default {
         this.loadingSubmission = false;
       }
     },
+    // Attaches attributes to <a> Link tags to open in a new tab.
+    attachAttributesToLinks() {
+      let simpleContentComponents = formioUtils.searchComponents(this.formSchema.components, {
```

Comment:
This should be a const - it's not being reassigned anywhere else in this function scope.

Reply:
Moving this fix into a new PR-196
https://github.com/bcgov/common-hosted-form-service/pull/196